presto-docs/src/main/sphinx/release/release-0.249.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.249
+=============
+
+**Highlights**
+==============
+* Add Support for Hive Connector to work with Hive 3 metastore for basic tables.
+* Update joda to 2.10.8 tzdata 2020d.
+* Support computing CRC32 checksum for data exchanges.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an error executing index joins when join spilling is enabled.
+* Enforce ``max_total_memory_per_node`` on user memory allocation.
+* Update joda to 2.10.8 tzdata 2020d. Make sure you deploy with a JVM that is using matching tzdata.
+* Support computing CRC32 checksum for data exchanges. Disabled by default and can be enabled by setting the `exchange.checksum-enabled=true` configuration property.
+
+Hive Changes
+____________
+* Add Support for Hive Connector to work with Hive 3 metastore for basic tables (ACID Tables and tables using constraints are not supported yet) (:pr:`15805`).
+* Fix a bug when S3 access is compromised across s3 enabled hive catalogs (:pr:`15800`).
+* Add `new_partition_user_supplied_parameter` session property which causes all partitions created by a query to have the `user_supplied` parameter set to the supplied string in Hive metastore.
+
+Presto on Spark Changes
+_______________________
+* Add support for distributing broadcast table using persistent storage, thereby removing spark driver from the distribution flow. The feature can be enabled/disabled using `storage_based_broadcast_join_enabled` session property.
```

Comment:
maybe add the configuration property too if there is one. Also, what's the default value?